tests/chart_tests/test_probes.py

```diff
@@ -26,6 +26,22 @@ def test_template_probes_with_custom_values(self, doc):
             assert container["livenessProbe"] != {}
             assert container["readinessProbe"] != {}
 
+    filtered_docs = [doc for doc in docs if doc["kind"] in include_kind_list]
+
+    @pytest.mark.parametrize("doc", filtered_docs)
+    def test_init_containers_have_no_probes(self, doc):
+        """Ensure init containers do not have liveness or readiness probes."""
+
+        spec = doc.get("spec", {})
```

Comment:
Maybe skip things that have no spec?
```suggestion
        if not (spec = doc.get("spec", {})):
            pytest.skip("doc has no .spec")
```